firmware/common2015/utils/logger/logger.cpp

```diff
@@ -70,3 +105,19 @@ int logLvlChange(const std::string& s) {
 
     return n;
 }
+
+void findSmallestLogFile() {
```

Comment:
Could you make this function return the logNameInd, and use it as a scoped variable above? It just seems kind of weird having it modify global variables.